### PR TITLE
ci(package-manager): replace use of soon-to-be removed ubuntu runner

### PR DESCRIPTION
**Pull request by Fdawgs**
`ubuntu-16.04` will be removed as a runner this year: [see GitHub Blog here](https://github.blog/changelog/2021-04-29-github-actions-ubuntu-16-04-lts-virtual-environment-will-be-removed-on-september-20-2021/).

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,17 +2,17 @@ name: Continuous Integration
...
-      - "*.md"
+      - '*.md'
 
 jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [15]
+        node-version: [15, 16]
```
Comment (kibertoad) [inline .github/workflows/ci.yml:15]: do we still need to run it on Node 15?

File: .github/workflows/package-manager-ci.yml [hunk 2]
```diff
@@ -2,8 +2,11 @@ name: package-manager-ci
 
 on:
   push:
-    branches:
-      - master
```
Comment (kibertoad) [inline .github/workflows/package-manager-ci.yml:6]: why is this no longer necessary?

Comment (Fdawgs) [reply .github/workflows/package-manager-ci.yml:6]: Whoops, thanks @kibertoad!

**Review by climba03003 — COMMENTED** [review]
I think we need to remove `.github/workflow/package-manager-ci.yml`, or remove node version `10,12,14` as those version always fail.

File: .github/workflows/package-manager-ci.yml [hunk 3]
```diff
@@ -42,7 +47,7 @@ jobs:
       matrix:
         # Maintenance and active LTS
         node-version: [10, 12, 14, 16]
```
Comment (climba03003) [inline .github/workflows/package-manager-ci.yml:49]:
```suggestion
        node-version: [16]
```

File: .github/workflows/package-manager-ci.yml [hunk 4]
```diff
@@ -13,7 +18,7 @@ jobs:
...
```
Comment (climba03003) [inline .github/workflows/package-manager-ci.yml:20]:
```suggestion
        node-version: [16]
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Fdawgs) [reply .github/workflows/ci.yml:15]: Engine in `package.json` is set at `>=15.0.0`, so... maybe?
I'm not keen on testing on non-LTS versions either, so if the general consensus is to get rid then I'm more than happy to remove.

Comment (climba03003) [reply .github/workflows/ci.yml:15]: Removing the test version means we drop support of that node version. I think we should keep that.

**Review by climba03003 — APPROVED** [review]
LGTM.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Fdawgs) [reply .github/workflows/ci.yml:15]: What do you think @kibertoad?

Comment (RafaelGSS) [reply .github/workflows/ci.yml:15]: `diagnostics_channel` was landed in v15. It might better to keep it.

Comment (kibertoad) [reply .github/workflows/ci.yml:15]: yeah, makes sense, better to keep it then

**Pull request merged**